com.unity.shadergraph/Editor/GraphUI/Importers/ShaderGraphImporterEditor.cs

```diff
@@ -13,7 +16,19 @@ class NewShaderGraphImporterEditor : ScriptedImporterEditor
         MaterialEditor materialEditor = null;
         public override void OnInspectorGUI()
         {
-            EditorGUILayout.HelpBox("WIP", MessageType.Info);
```

Comment:
You could put the function scope or block scope here as a 'preview of the preview shader'.

Reply:
I will post that question to Design and an update can happen in another PR if needed.